cmd/runner/main.go

```diff
@@ -15,26 +15,41 @@ package main
 
 import (
 	"flag"
+	"fmt"
+	"io/ioutil"
 	"log"
 	"time"
 
 	"github.com/grpc/test-infra/tools/runner"
 )
 
+// defaultOutputSuiteName provides a default name for the testsuites tag
+// in an XML report. It is based on the number of nanoseconds since the
+// UNIX epoch.
+var defaultOutputSuiteName = fmt.Sprintf("benchmarks-%d", time.Now().UnixNano())
+
 func main() {
 	var i runner.FileNames
+	var o runner.FileNames
```

Comment:
If we know we are generating only a single report, then this argument can be a string instead of a list accumulator.

If we want ot specify multiple reports, then we may want to have a map accumulator instead, as we have for concurrency levels. I might rename FileNames to InputFileNames and call the new map accumulator OutputFileNames.

Reply:
Right, I temporarily used a runner.FileNames since it provides validation. In this case, it requires the string to not be empty. We could create a OutputFileName flag type that validates the location is writable on the flag parse Set call. We could also validate that the files exist for FileNames, making it InputFileNames. Or we can choose to handle the logic later. The flexibility of the flag.Var interface is quite nice.